registry/scenarios.go

```diff
@@ -3,23 +3,34 @@ package registry
 import (
 	"strings"
 
+	dockervm "github.com/DataDog/test-infra-definitions/aws/scenarios/dockerVM"
+	"github.com/DataDog/test-infra-definitions/aws/scenarios/ecs"
+	"github.com/DataDog/test-infra-definitions/aws/scenarios/eks"
+	"github.com/DataDog/test-infra-definitions/aws/scenarios/microVMs/microvms"
+	"github.com/DataDog/test-infra-definitions/azure/scenarios/aks"
+	azvm "github.com/DataDog/test-infra-definitions/azure/scenarios/vm"
+
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
 )
 
-type scenarioRegistry map[string]pulumi.RunFunc
-
-var Scenarios = make(scenarioRegistry)
+type ScenarioRegistry map[string]pulumi.RunFunc
 
-// Register lower case the given name
-func (s scenarioRegistry) Register(name string, rf pulumi.RunFunc) {
-	s[strings.ToLower(name)] = rf
+func Scenarios() ScenarioRegistry {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
@ogaca-dd what is the advantage of registering scenarios here, rather than have each scenario registering itself independently ? 

I see the cons of this implementation as leading to possibly forgetting scenarios, as adding one requires editing 2 files instead of just one - the scenario `run`. 
Another cons of this implementation is that we would edit the same file when adding scenarios, leading to possible git conflicts.